fernando-gomez/challenge/pokeapi/app/components/pokeList/pokeList.js

```diff
@@ -0,0 +1,69 @@
+const main = document.querySelector('.main-container');
+const DOMpokeList = document.querySelector('.pokeList');
+let offset = 0;
+const limit = 70;
+
+const pokeElement = (tag, data) => {
+  const element = document.createElement(tag);
+  element.innerText = data;
+  return element;
+};
+
+const pokemonAnchor = (pokemon) => {
+  const anchor = document.createElement('a');
+  anchor.setAttribute('href', `./../pokeDetail/pokeDetail.html?name=${pokemon.name}`);
+
+  return anchor;
+};
+
+const removeAllChildNodes = (parent) => {
+  while (parent.firstChild) {
+    parent.removeChild(parent.firstChild);
+  }
```

Comment:
Un while es como el hijo feo de los loop. Intenta siempre, primero comprobar si existe un parent y si tiene hijos, y en caso afirmativo, bórralos con un for.